### PR TITLE
chore!: Switch corq to dorq

### PR DESCRIPTION
**Pull request by alexjuda**
https://zapatacomputing.atlassian.net/browse/ORQSDK-748

# The problem

Users won't use our new CLI unless we expose it under a nice alias.

# This PR's solution

* Point `orq` to dorq CLI
* Fix CLI argument wording
* Adopt CLI reference doc for click

# Checklist

_Check that this PR satisfies the following items:_

- [x] Tests have been added for new features/changed behavior (if no new features have been added, check the box).
- [x] The [changelog file](CHANGELOG.md) has been updated with a user-readable description of the changes (if the change isn't visible to the user in any way, check the box).
- [x] The PR's title is prefixed with `<feat/fix/chore/internal/docs>[!]:`

### CODE REVIEW

File: tests/runtime/performance/test_cli_perf.py [hunk 1]
```diff
@@ -11,8 +11,10 @@
 https://zapatacomputing.atlassian.net/browse/ORQSDK-507
```
Comment (jamesclark-Zapata) [inline tests/runtime/performance/test_cli_perf.py:11]: **question:** ‏Are these relevant now? Should they be switched to the new CLI?

**Review by jamesclark-Zapata — APPROVED** [review]
question about perf test, but otherwise lgtm

**Review by jamesclark-Zapata — APPROVED** [review]
question about perf test, but otherwise lgtm

File: tests/runtime/performance/test_cli_perf.py (hunk 1 above)
Comment (alexjuda) [reply tests/runtime/performance/test_cli_perf.py:11]: They do test that corq still works. I think we should keep them as long as we keep corq code.

Comment (alexjuda) [reply tests/runtime/performance/test_cli_perf.py:11]: > Should they be switched to the new CLI?

Having similar ones for dorq sounds like a good idea, but I'd keep it out of scope. Added under https://zapatacomputing.atlassian.net/browse/ORQSDK-749

**Pull request merged**